x-pack/plugins/lens/public/shared_components/toolbar_popover.tsx

```diff
@@ -11,7 +11,8 @@ import { EuiIconLegend } from '../assets/legend';
 
 const typeToIconMap: { [type: string]: string | IconType } = {
   legend: EuiIconLegend as IconType,
-  values: 'number',
+  values: 'visText',
+  numeric: 'number',
```

Comment:
I think "numeric" is a confusing name for this. Let's go with "values" (for number) and "labels" (for visText) as this is what's used for the title of the popover as well so we stay consistent with the UI:
<img width="404" alt="Screenshot 2020-11-09 at 15 56 58" src="https://user-images.githubusercontent.com/1508364/98557290-502c6600-22a4-11eb-8a32-dd2b0879031a.png">
<img width="359" alt="Screenshot 2020-11-09 at 15 57 03" src="https://user-images.githubusercontent.com/1508364/98557300-51f62980-22a4-11eb-97c2-374b60e45db5.png">